pages/witch-pricker.vue

```diff
@@ -2,32 +2,17 @@
   <div>
     <div id="main-content" class="pl-5 pr-5 pt-3 pb-3">
       <div class="flex content-start items-center">
-        <h1 class="text-sm md:text-xl lg:text-2xl">
-          The Journey of a Witch-Pricker - John Kincaid of Tranent
-        </h1>
-        <span
-          class="rounded-full border-r border-l border-gray-400 w-6 h-6 flex items-center justify-center ml-2"
+        <h1>The Journey of a Witch-Pricker - John Kincaid of Tranent</h1>
+        <div
+          class="inline-flex items-center justify-center align-middle rounded-full border-r-2 border-l-2 border-gray-400 w-6 h-6 hover:w-7 hover:h-7 mb-0.5 ml-1 cursor-pointer"
         >
-          <!-- icon by feathericons.com -->
-          <svg
-            aria-hidden="true"
-            class=""
-            data-reactid="266"
-            fill="none"
-            height="24"
-            stroke="#606F7B"
-            stroke-linecap="round"
-            stroke-linejoin="round"
-            stroke-width="2"
-            viewBox="0 0 24 24"
-            width="24"
-            xmlns="http://www.w3.org/2000/svg"
+          <img
+            src="/images/infoIcon.svg"
+            class="w-full h-full pt-0.5"
+            alt="Page Information Popup"
             @click="showPageInfo()"
```

Comment:
Just adding this here as I've got a bit lost as to where my original comments were. I know you changed the icon for the info popup and made it have a hand cursor. I did reply to that change but I can't find which branch it was in now. Would it be possible to change the icon to use the feather icon info instead?

Reason being, it looks more like an information icon. The current one doesn't really look like an icon. It just looks like an i in brackets so it isn't immediately obvious it isn't just part of the text. I think we should also change the icon colour to make it pop more. A blue colour might be appropriate here.

Finally we should make sure the icon is accessible for folks who will be using a screenreader. Usually this is achieved by adding aria lables to the icon and also having a visually hidden bit of text that will only be read by a screenreader but not shown on the page. Thanks Ruby

Reply:
done, have change the infoIcon to be the feather icon you suggested and it is blue. have added aria-label and title so is screen readable

Reply:
Thanks Ruby. I think this looks much better so cheers for changing this. Only concern I have is that the styling isn't quite right on some pages. See the screenshot below for the main page:

![image](https://github.com/user-attachments/assets/36610a56-1652-4939-991b-f1d1af363e41)

Reply:
Sorry Ruby could you also point me in the direction of the aria-label and hidden text? I can't spot it in the code. Thank you

Reply:
![image](https://github.com/user-attachments/assets/48a478cc-a31b-472f-8fee-36ddedd85af3)
This is what it looks like on my screen. If I zoom out I do have the same issue as you although I'm not sure how to fix it so that it is always in the same line even if there is no room for it. any suggestions unless I make the text slightly bigger?

Reply:
Could we use a non breaking space e.g. `&nbsp;` instead of white space at the end of the text?

Reply:
and thanks for fixing the visually hidden stuff. Looks good to me.

Reply:
as its a passed in parameter from each page with the filter and map component adding &nbsp; just adds 2 spaces, the passed in parameters also don't have spaces a the end. If i changed it to h1 then this doesn't happen, should i do this? 
![image](https://github.com/user-attachments/assets/f77991fc-71b7-4c3c-a9ee-4efdfa7daec0) If using h1 should i change the font weight so not as bold?

Reply:
ok thanks Ruby. I wasn't entirely sure how to fix this without playing about with the code. I've kept it as a h1 but adjusted the size so it isn't too big for the drawer.

I've changed so we are using flex box to align the icon to the right always. It isn't perfect but I think will suffice for launch. 

I've pushed the changes now. Please could you take a look and if you are happy, I think we can mark this as closed